Moderation-Policy.md

```diff
@@ -202,6 +202,8 @@ of the [Code of Conduct][].
   * report accounts to GitHub
 * Accounts that are reasonably believed to be bots (other than bots authorized
   by both TSC and CommComm) are subject to immediate Blocking.
+* Issues, pull requests, discussions and comments that are considered spam by the Moderation Team
+  (such as job posting, service advertising, etc.) will be deleted.
```

Comment:
For consistency with the other items in the list:

```suggestion
* Issues, pull requests, discussions, and comments that are spam (job posting,
  service advertising, etc.) are subject to Immediate moderation.
```

Reply:
lower case "immediate"?

Reply:
> lower case "immediate"?

Yes. Fixed. Thanks!